Source/cmake/OptionsGTK.cmake

```diff
@@ -407,6 +408,19 @@ if (USE_WOFF2)
     endif ()
 endif ()
 
+if (ENABLE_WEBXR)
+    if (NOT ENABLE_GAMEPAD)
+        message(FATAL_ERROR "Gamepad is required to be enabled for WebXR support.")
+    endif ()
+    find_package(OpenXR 1.0.20)
+    if (NOT OPENXR_FOUND)
+        message(FATAL_ERROR "OpenXR is required to enable WebXR support.")
+    endif ()
+    SET_AND_EXPOSE_TO_BUILD(USE_OPENXR ${OpenXR_FOUND})
+    SET_AND_EXPOSE_TO_BUILD(XR_USE_PLATFORM_EGL TRUE)
+    SET_AND_EXPOSE_TO_BUILD(XR_USE_GRAPHICS_API_OPENGL_ES TRUE)
```

Comment:
Why do we need these if they are always TRUE?

Reply:
Because we'd need different defines for other platforms, for example Android